params/others/Run_topmodel.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Aug 13 12:51:06 2021
+
+@author: west
+"""
+
+import subprocess
+import os
+import glob
+import shutil
+import pandas as pd
+import matplotlib.pyplot as plt   
+import sys  
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(os.getcwd())))
+sys.path.append(BASE_DIR+"/params/src/")
+import summarize_results_functions as FC
+
+outputfolder_results=BASE_DIR+"/params/data/TopModel_Results/"
+if not os.path.exists(outputfolder_results): os.mkdir(outputfolder_results)
+
+run_flag=1
+process_results_flag=1
+Resolution=30    
+#hydro_fabrics_input_dir="/home/west/Projects/hydrofabrics/20210511/"
```

Comment:
OWP guidelines say we cannot include paths that reveal a computer's directory structure. Please remove.